evmjit/libevmjit/Compiler.cpp

```diff
@@ -49,6 +49,13 @@ void Compiler::createBasicBlocks(code_iterator _codeBegin, code_iterator _codeEn
 		return _curr + offset;
 	};
 
+	// Skip all STOPs in the end
+	for (; _codeEnd != _codeBegin; --_codeEnd)
+	{
```

Comment:
Not that one again :)
Braces are not needed only if they contain one line of code. These contain two.